ieeeuoft/hackathon-template
# IEEE-202: layout for orders page and code cleanup

## Overview

- Resolves #IEEE-202
- Made layout for orders page 
- modified some css

![image](https://user-images.githubusercontent.com/20618460/174689477-1aef311e-67a7-403b-b74c-1e86f1bd593f.png)



## Unit Tests Created

- non added


## Steps to QA

- go to orders page and resize screen to different widths and heights, make sure everything displays clearly
- the filters component should disappear for smaller screen widths
